test/e2e/catalog_e2e_test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"time"
 
 	operatorsv1 "github.com/operator-framework/api/pkg/operators/v1"
+	k8serror "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
nit: it would be nice to standardize import naming conventions at some point. This can be enforced by introducing more configuration through golangci-lint's importas linter like we currently do with rukpak.

Reply:
Maybe we could add a ticket to the CI epic?

Reply:
Yep - sounds reasonable to me.

Reply:
Updated the Epic and added a ticket for this. Opened https://github.com/operator-framework/operator-lifecycle-manager/pull/2715 since then that tackles that kind of linting enforcement.